mmv1/products/redis/Cluster.yaml

```diff
@@ -276,3 +276,121 @@ properties:
       Configure Redis Cluster behavior using a subset of native Redis configuration parameters.
       Please check Memorystore documentation for the list of supported parameters:
       https://cloud.google.com/memorystore/docs/cluster/supported-instance-configurations
+  - !ruby/object:Api::Type::NestedObject
+    name: maintenancePolicy
+    description: Maintenance policy for a cluster
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'createTime'
+        output: true
+        description: |
+          Output only. The time when the policy was created.
+          A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+          resolution and up to nine fractional digits.
+      - !ruby/object:Api::Type::String
+        name: 'updateTime'
+        output: true
+        description: |
+          Output only. The time when the policy was last updated.
+          A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+          resolution and up to nine fractional digits.
+      - !ruby/object:Api::Type::Array
+        name: 'weeklyMaintenanceWindow'
+        description: |
+          Optional. Maintenance window that is applied to resources covered by this policy.
+          Minimum 1. For the current version, the maximum number
+          of weekly_window is expected to be one.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Enum
+              name: 'day'
+              required: true
+              description: |
+                Required. The day of week that maintenance updates occur.
+
+                - DAY_OF_WEEK_UNSPECIFIED: The day of the week is unspecified.
```

Comment:
```suggestion
```

We usually omit these values from enums, as they aren't valid for users to supply

Reply:
We have taken reference from https://github.com/GoogleCloudPlatform/magic-modules/blob/692bae292bf8c33f8443e60cc53bab800b4821a2/mmv1/products/redis/Instance.yaml#L316

and want to keep the Cluster changes similar to Redis Instance.
Is this fine? or should I remove this from the Cluster.yaml